src/redux/reducers.js

```diff
@@ -1233,3 +1361,18 @@ export function usernameSubscribers(state = {}, action) {
 export function usernameSubscriptions(state = {}, action) {
   return handleSubs(state, action, ActionTypes.SUBSCRIPTIONS)
 }
+
+export function compensateScroll(state = {}, action) {
```

Comment:
`scrollCompensation` then, maybe? It's just the only one substate with a verb instead of noun. But if you really have a strong opinion on it, let it be.